src/app/shared/quick-control/quick-control.component.scss

```diff
@@ -0,0 +1,57 @@
+.quick-control {
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.85);
+  // backdrop-filter: blur(8px);
+  transition: opacity 0.4s ease-in-out;
+  z-index: 10;
+  opacity: 1;
```

Comment:
If set to 0 as it was before extracting the component, i could not make it appear with the js calls to change opacity with a transition effect